CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-## v1.3.1
+## v1.4.1
+
+- Add chromosome grid and label overlays directly from cooler files
```

Comment:
Didn't we add the chrom grid in v1.4?

Reply:
What I meant to say is that chromsizes can be pulled directly from cooler files. This mostly just adds that as an option to the Add Track Dialog.

Reply:
I'll change the message.